projects/storefrontlib/src/layout/loading/index.ts

```diff
@@ -1,3 +1,3 @@
 export { DeferLoaderService as θDeferLoaderService } from './defer-loader.service';
 export { IntersectionOptions as θIntersectionOptions } from './intersection.model';
```

Comment:
can we make options and loader service public as well?

Reply:
Done